internal/app/dogma/watch_cmd.go

```diff
@@ -55,37 +103,32 @@ func (wc *watchCommand) execute(c *cli.Context) error {
 	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
LGTM. 

Just a note that is not related to the PR:
- By looking at this file carefully, I think `ctx` here is no need.
- Done channel could be set to size 1 instead of 2. `1` sounds more `convention` to me.

Reply:
> I think ctx here is no need.

Exactly.

> 1 sounds more convention to me.

To me too.

I'll send another PR after this #42 gets merged.